tests/PHPUnit/System/expected/test_CustomEvents_eventCategoryOrNameMatch__Events.getName_day.xml

```diff
@@ -7,7 +7,7 @@
 		<nb_events>16</nb_events>
 		<nb_events_with_value>2</nb_events_with_value>
 		<sum_event_value>19.32</sum_event_value>
-		<min_event_value>9.66</min_event_value>
+		<min_event_value>0</min_event_value>
```

Comment:
This one somehow still does not look correct:

nb_events_with_value: 2
sum_event_value: 19.32
min_event_value: 0
max_event_value: 9.66
avg_event_value: 9.66

_This should either be:_
nb_events_with_value: 2
sum_event_value: 19.32
min_event_value: 0
max_event_value: **> 19.32 <**
avg_event_value: 9.66

_Or:_
nb_events_with_value: 2
sum_event_value: 19.32
min_event_value: 9.66
max_event_value: 9.66
avg_event_value: 9.66

Reply:
`plugins/Events/RecordBuilders/EventReports.php` needs a change:

https://github.com/matomo-org/matomo/blob/bebbd31bf7253293e71b493e88fb859939bca34e/plugins/Events/RecordBuilders/EventReports.php#L195-L203

- `php -r "echo round(null ?? 0, 2);"` returns `0`
- `php -r "echo round(null ?? false, 2);"` returns `0`
- `php -r "echo round('' ?? 0, 2);"` returns `0`
- `php -r "echo round('' ?? false, 2);"` also returns `0`

To return `false` this should become:

```php
Metrics::INDEX_EVENT_MIN_EVENT_VALUE    => is_numeric($row[Metrics::INDEX_EVENT_MIN_EVENT_VALUE]) ? round($row[Metrics::INDEX_EVENT_MIN_EVENT_VALUE], 2) : false,
```

Reply:
good catch, will update that